_shows/01_02/cabaret.md

```diff
@@ -11,5 +11,8 @@ date_end: 2001-12-08
 crew:
  - role: Musical Director
    name: Robert Wicks
+ - role: Band, Trombone
```

Comment:
Can this be
```
role: Band
note: Trombone
```
Which I think then matches up with our other formats, as I don't think there's a programme formatted in that way for this show.